[2024-10-12T19:27:38.207+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-12T19:27:38.220+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow.get_age scheduled__2024-10-10T00:00:00+00:00 [queued]>
[2024-10-12T19:27:38.226+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow.get_age scheduled__2024-10-10T00:00:00+00:00 [queued]>
[2024-10-12T19:27:38.227+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-12T19:27:38.237+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_age> on 2024-10-10 00:00:00+00:00
[2024-10-12T19:27:38.242+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow', 'get_age', 'scheduled__2024-10-10T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/taskflow.py', '--cfg-path', '/tmp/tmph3hqo8su']
[2024-10-12T19:27:38.244+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1621) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-12T19:27:38.245+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask get_age
[2024-10-12T19:27:38.245+0000] {standard_task_runner.py:72} INFO - Started process 1624 to run task
[2024-10-12T19:27:38.274+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow.get_age scheduled__2024-10-10T00:00:00+00:00 [running]> on host 91a8a8c9b949
[2024-10-12T19:27:38.323+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='james' AIRFLOW_CTX_DAG_ID='dag_with_taskflow' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2024-10-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-10T00:00:00+00:00'
[2024-10-12T19:27:38.326+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-12T19:27:38.329+0000] {python.py:240} INFO - Done. Returned value was: 29
[2024-10-12T19:27:38.345+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-12T19:27:38.346+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow, task_id=get_age, run_id=scheduled__2024-10-10T00:00:00+00:00, execution_date=20241010T000000, start_date=20241012T192738, end_date=20241012T192738
[2024-10-12T19:27:38.384+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-12T19:27:38.399+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-12T19:27:38.400+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
